src/OpenDiffix.Core/Analyzer.fs

```diff
@@ -53,8 +55,15 @@ let private mapFunctionExpression rangeColumns fn parsedArgs =
          { aggregateArgs with Distinct = true }, [ mapExpression rangeColumns parsedExpr ]
        | parsedExpr -> aggregateArgs, [ mapExpression rangeColumns parsedExpr ]
 
-     let aids = parsedAids |> List.map (mapExpression rangeColumns) |> ListExpr
-     AggregateFunction(DiffixCount, aggregateArgs), aids :: args
+     AggregateFunction(DiffixCount, aggregateArgs), mapAids parsedAids :: args
+   | AggregateFunction (DiffixSum, aggregateArgs), parsedArg :: parsedAids ->
+     let aggregateArgs, args =
+       match parsedArg with
+       | ParserTypes.Distinct parsedExpr ->
+         { aggregateArgs with Distinct = true }, [ mapExpression rangeColumns parsedExpr ]
```

Comment:
Do we reject `sum(DISTINCT x)`?

Reply:
yes, but done in `QueryValidator` along with `sum(something_complex)`